person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
+  end
+
+  private :is_of_age?
```

Comment:
- [ ] The `:is_of_age?` method does not exist in this class. Kindly fix that. I think you meant `:of_age?`.